src/hooks/useIsMobileWidth.ts

```diff
@@ -0,0 +1,16 @@
+import { useEffect, useState } from 'react'
+
+const MOBILE_BREAKPOINT_WIDTH = 900
```

Comment:
breakpoints should go in a theme file so they stay consistent throughout the codebase.
Can you move this into a new src/theme/breakpoints.ts file? It can be an enum imo, but up to you (eg `enum BREAKPOINTS { MOBILE = 900 }`).